visualizations/load-aware/plotting/pod_times.py

```diff
@@ -168,7 +179,38 @@ def do_plot(self, ordered_vars, settings, setting_lists, variables, cfg):
         title = f"Pod execution timeline"
 
         fig.update_layout(title=title, title_x=0.5)
-
+       
+        scheduling_phase = df.where(df["Phase"] == "Scheduling")
```

Comment:
```suggestion
        scheduling_phase = df[df.Phase == "Scheduling"]
```
wasn't straightforward to troubleshoot :D

Reply:
```
                   Start                 End            Creation  Duration                    Pod                                        Node Workload       Phase
0    2023-07-24 16:39:41 2023-07-24 16:39:41 2023-07-24 16:39:41       0.0  coreutils-make-pod-n0  ip-10-0-142-236.us-west-2.compute.internal     make  Scheduling
1                    NaT                 NaT                 NaT       NaN                    NaN                                         NaN      NaN         NaN
2                    NaT                 NaT                 NaT       NaN                    NaN                                         NaN      NaN         NaN
601                  NaT                 NaT                 NaT       NaN                    NaN                                         NaN      NaN         NaN
```